Gali 825

shyogesh-sw opened the pull request:
Issue #, if available:

Description of changes: Updated the screenshots as per Gali-825

Checklist:

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?

<!-- For major releases please provide internal ticket id -->

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

shyogesh-sw committed:
<docs: added hyperlinks in packer readme>

shyogesh-sw committed:
docs: updated the screenshots gali825

shyogesh-sw committed:
Merge branch 'develop' into gali-825

jn1119 reviewed (commented):
LGTM, do we need to delete old snapshots sc_ec2_windows_00.png and sc_ec2_windows_01.png?

jn1119 committed:
Merge branch 'develop' into gali-825

shyogesh-sw committed:
docs: removed old screenshots from Git

shyogesh-sw committed:
docs: deleted old screenshots

Merge branch 'gali-825' of https://github.com/shyogesh-sw/service-workbench-on-aws into gali-825

shyogesh-sw commented:
> 
> 
> LGTM, do we need to delete old snapshots sc_ec2_windows_00.png and sc_ec2_windows_01.png?

Yogesh: I have deleted the two screenshots from Git and updated the PR.
